extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts

```diff
@@ -446,6 +515,14 @@ export class MigrationCutoverDialog {
 		return header;
 	}
 
+	private setAutoRefresh(interval: SupportedAutoRefreshIntervals): void {
```

Comment:
Consider making a base class for common functionality like this so you don't have to repeat this function for every page that uses it.